karma.conf.js

```diff
@@ -20,7 +20,7 @@ module.exports = function (config) {
       'karma-sourcemap-loader'
     ],
 
-    browsers: ['PhantomJS'],
+    browsers: process.env.CI === true ? ['PhantomJS'] : [],
```

Comment:
If they specify a browser, it will just be a single-run process. 

By not specifying a browser, it allows devs to use whichever browser they want to debug the tests.